packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -123,6 +122,17 @@ const seedlessOnboardingMetadata: StateMetadata<SeedlessOnboardingControllerStat
       persist: false,
       anonymous: true,
     },
+    // stays in vault
+    accessToken: {
+      persist: false,
+      anonymous: true,
+    },
+    // stays outside of vault as this token is accessed by the metadata service
+    // before the vault is created or unlocked.
+    metadataAccessToken: {
+      persist: false,
+      anonymous: true,
+    },
```

Comment:
is it correct that this item should **not** be persisted even though it lives outside the vault?